python/oneseismic/scan/scanners.py

```diff
@@ -309,6 +314,8 @@ def scan_trace_header(self, header):
 
 
     def report(self):
+        # Use the 3 corner points (or the points closest to the corners) that
+        # was found during the scan to find a mapping from line numbers to cdp
```

Comment:
I have another idea for the representation of the results from this scanner that I think will be easier to draw up and explain. I think this will be easier to review as a new PR though. So remove it for now and pinky promise that better explanations are coming very soon?